src/components/ControllerTop.vue

```diff
@@ -238,8 +238,7 @@ export default {
             this.updateLayout(data.layout);
             let promise = new Promise(resolve =>
               store.commit('keymap/setLoadingKeymapPromise', resolve)
```

Comment:
Waiting for a commit wouldn't be in an action? it's more the vue design?

Reply:
this isn't an action, it's a signaling mechanism.